app/src/main/res/values-fr/strings.xml

```diff
@@ -0,0 +1,375 @@
+<resources>
+    <string name="app_name" translatable="false">monerujo</string>
+    <string name="login_activity_name" translatable="false">monerujo</string>
```

Comment:
please delete all strings with ```translatable="false"``` - they may only be in the original files.

Reply:
also, i think you may be working off an old version. please rebase to current master.

Reply:
ok, done.